features/step_definitions/api_request_steps.rb

```diff
@@ -261,6 +271,16 @@
   }]
 end
 
+Given("I add {string} which are part of multiple regular_transactions, of {float} per month of credit") do |category, amount|
```

Comment:
this title is unclear not sure that `which are part` adds to the clarity, the title of the step definition above is good and if you do not unify these into a single step definition then it may be worth just replicating the title above but switching `credit` to `debit`